src/core/utils/helpers/modal-helpers.ts

```diff
@@ -0,0 +1,54 @@
+import { getModalIds } from "./general";
+
+export const containsDueDateModalString = (modalString: string) => {
```

Comment:
Not fond of using the type of variable as a part of the variable. Something that describes the context of the modalId would clarify better what the purpose of the function is, I think.